rfcs/load_test.md

```diff
@@ -0,0 +1,46 @@
+# RFC 33: Crash Tests
+
+## Summary
+
+Add a test type in which the test document is simply loaded. The test
+passes if this occurs without encountering a fatal error e.g. a crash.
+
+## Details
+
+Load Tests are static tests typically used for cases where browsers
+are known to have a bug such as a crash or a leak. They differ from
+other tests such as reftests and testharness tests in that there is no
+pass condition other than the page loading successfully. In vendor
+infrastructure these tests may also be run with additional
+instrumentation e.g. address sanitizing, making the pass condition
+more stringent. The lack of an explicit pass condition makes it easier
+to write these tests than tests where a detailed understanding of the
+correct behaviour is required, making them suitable for autogeneration
+e.g. as the output of a fuzzing tool.
+
+The infrastructure changes are as follows:
+
+* A new test type `crash` recognised by the manifest and assigned to
+  any test file with a name containing `-crashtest` immediately before
+  the extension or in a subdirectory called `crashtests`.
+
+* A new executor type with the following behaviour:
+  - Load the test URL
+  - If there is no `class=wait` on the root element, mark the test as
```

Comment:
Will we want to generalize `class=wait` to reftests in future?

Reply:
That seems like a possibility, but I don't think it affects this PR.